### DISCUSSION TITLE
Any working sample of displaying 3d model in a SceneView

### DISCUSSION THREAD

**Discussion by depindersharma** [sceneview/sceneform-android#3] [Q&A]
Is there any sample available where the local 3d model can be displayed in the SceneView?
I was trying to use this but seems like the SceneView is not working. (it always return a black screen)

**Comment by mickod** [comment]
The following is an example that builds a model from a local GLB file and it's tested and working:

https://stackoverflow.com/a/65522584/334402

**Reply by depindersharma** [reply]
getFileFromAssets() , I am using java. Is this the custom function you have written? @mickod

**Comment by Wrakor** [comment]
I'm having the same issue and also would like a sample for SceneView. Did you have success?

**Comment by ThomasGorisse** [comment]
See the [SceneView Sample](https://github.com/ThomasGorisse/sceneform-android-sdk/tree/master/samples/sceneview-background)
